lib/cisco_node_utils/feature.rb

```diff
@@ -112,6 +112,13 @@ def self.nv_overlay_enable
       config_set('feature', 'nv_overlay')
     end
 
+    def self.nv_overlay_disable
+      # Note: vdc platforms restrict this feature to F3 or newer linecards
+      # Note: this is for test purposes only
+      return unless nv_overlay_enabled?
+      config_set('feature', 'nv_overlay', state: 'no')
```

Comment:
`nv_overlay` does not currently support the `state` keyword.

Add `<state>` [to the `set value` entry](https://github.com/cisco/cisco-network-node-utils/pull/341/files#diff-96c7c3f23af838d91015cff4d91c8501L39)